tests/test_blog_api.py

```diff
@@ -48,3 +49,5 @@ def test_get_articles_without_transforming_links(self):
             "admin.insights.ubuntu.com/wp-content/uploads",
             article["content"]["rendered"],
         )
+
+        self.assertNotIn("cloudinary", article["content"]["rendered"])
```

Comment:
Not sure about this assert here. There is no link to transform, but `cloudinary` could still be in the article content even if there are no trnasformed links.

Reply:
Yes, given that the tests should be running against fixtures anyway, we should be able to pick out a specific image URL in that article that we expect to have been, or not to have been, transformed, and check the state of that specific URL.